app/frontend/wallet/shelley/shelley-js-crypto-provider.ts

```diff
@@ -48,6 +48,8 @@ CryptoProviderParams): Promise<CryptoProvider> => {
 
   const getDerivationScheme = () => derivationScheme
 
+  const getVersion = () => derivationScheme.type
```

Comment:
I think coupling the derivation scheme with "version" is unnecessarily misleading. Technically hw wallets use derivation scheme v2 so I'd decouple that and submit the derivation scheme number and wallet version separately, and it's ok to return explicitly "null" for mnemonic-based wallets - at least it's more faithful to reality

Reply:
I'd perhaps even call this "getDeviceVersion" to make it clear this applies to devices

Reply:
fixed